apps/passport-server/src/services/telegramService.ts

```diff
@@ -849,8 +850,18 @@ export class TelegramService {
           newTimestamps
         );
       } else {
+        const nullifierTimeLeft =
+          new Date(
+            nullifierData.message_timestamps[
+              nullifierData.message_timestamps.length - 1
+            ]
+          ).getTime() +
+          rlDuration -
+          Date.now();
         throw new Error(
-          `You have exceeded the daily limit of ${maxDailyPostsPerTopic} messages for this topic.`
+          `You have exceeded the daily limit of ${maxDailyPostsPerTopic} messages for this topic. \n\nTry posting in another topic or wait ${msToTimeString(
```

Comment:
suggestion: maybe we shouldn't be so explicit with the rate limit here, since it's intended to stop spammers. We can just say "you have exceeded the daily limit for this topic".